docs/course_materials/compound_data_types.html

```diff
@@ -107,60 +108,33 @@ <h3><b>Popping an element (remove it and return it's value):</b></h3>
                 </section>
                 <section>
                     <h3><b>More actions</b></h3>
-                    <pre><code data-trim data-noescape class="hljs">
-lst[i] = v    # Changes an element or slice by assigning
-lst.extend(l) # Adds items from iterable
-lst.remove(v) # Removes a specific value
+                    <pre><code data-trim data-noescape class="hljs lang-python">
+lst[i] = v      # Changes an element or slice by assigning
+lst.remove(v)   # Removes a specific value
                     </code></pre>
-                </section>
-                <section>
-                    <h3><b>Sequence operations</b></h3>
                     <ul>
-                        <li>All sequence operations are valid on lists:</li>
+                        <li>The .extend() method adds all the elements of an iterable to the end of the list, and returns None:</li>
```

Comment:
Since they didn't learn about functions yet, I think it's better to skip the "and returns none"